react/components/Shipping/ShippingHeader.tsx

```diff
@@ -39,7 +41,17 @@ const ShippingHeader: FunctionComponent<Props & InjectedIntlProps> = ({
         <br />
         <small className="c-muted-2 t-small">{shippingData.selectedSla}</small>
       </p>
-      <Address address={shippingData.address} />
+      {giftRegistry &&
+      giftRegistry.addressId === shippingData.address.addressId ? (
```

Comment:
i see you are checking if `giftRegistry` exists. is `shippingData...` guaranteed to exist by this point?

Reply:
Yep, some of them could be `null`, but they are not `undefined`, so nothing should break.
My interface definition for `Parcel`, which is the type of `shippingData`:

```
interface Parcel {
  address: Address
  price: number
  pickupFriendlyName: string | null
  seller: string
  items: OrderItem[]
  selectedSla: string
  selectedSlaObj: ShippingSLA
  shippingEstimate: string | null
  shippingEstimateDate: string | null
  deliveryWindow: DeliveryWindow | null
  deliveryChannel: string
  selectedSlaType: string
}
```